lib/inject.js

```diff
@@ -140,6 +138,25 @@ function injectPa11y(window, options, done) {
 		return false;
 	}
 
+	function hiddenAreasContainsElement(element, hiddenSelectors) {
+		var hiddenElements = window.document.querySelectorAll(hiddenSelectors);
+
+		for(var i = 0; i < hiddenElements.length; i++) {
+			if(element.isEqualNode(hiddenElements[i])) {
```

Comment:
If it's fast enough with most sites then I think it's OK :) we can always refactor if performance is an issue.